modules/data.remote/R/call_MODIS.R

```diff
@@ -111,11 +166,39 @@ call_MODIS <- function(outfolder = ".", start_date, end_date, lat, lon, size = 0
     output$lat <- round(output$lat, 4)
     output$lon <- round(output$lon, 4)
     
-    fname <- paste(product, "_", band, "output_", start_date, "_", end_date, "_", lat, "_", lon, ".csv", sep = "")
-    fname <- paste0(outfolder, "/", fname)
-    write.csv(output, fname)
-    return(output)}
-
+    if (QC_filter == T)
+    {
+      output$qc == as.character(output$qc)
+      for (i in 1:nrow(output))
+      {
+        convert = paste(binaryLogic::as.binary(as.integer(output$qc[i]), n = 8), collapse = "")
+        output$qc[i] = substr(convert, nchar(convert)-2, nchar(convert))
```

Comment:
Note my earlier comment about `[` vs. `[[`. Also, please use `<-` for assignment throughout this block.

Reply:
output is a data.frame so I don't think i need doubles..... right???? just checking since this is new.